.github/workflows/integration.yml

```diff
@@ -39,15 +36,22 @@ jobs:
                   node-version: '20.12.2'
                   cache: 'yarn'
 
+            # Needed for local browser integration tests
+            # chrome-version documentation can be found here: https://github.com/browser-actions/setup-chrome?tab=readme-ov-file#usage
+            - name: Setup chrome v124
+              uses: browser-actions/setup-chrome@v1
+              with:
+                  chrome-version: 124
+              id: setup-chrome
+
```

Comment:
I'm using a slightly older Chrome version (124) just to prove that we are really using Chrome v124 and can pin it. If we try to use "latest," then invariable we'll get broken when there's a Chrome update.

Reply:
Are we just going to stick with Chrome 124 forever?

Reply:
For the time being, sure.

The alternative is being broken every 6 weeks 😛